samples/browser/Sample.html

```diff
@@ -60,36 +63,43 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
         </tr>
     </table>
 
+    <!-- The SDK has a dependency on requirejs (http://requirejs.org/). -->
+    <script src="samples\require.js"></script>
```

Comment:
We neednot include it in our package. The consumer of the sdk can choose how to reference it.